dist/less/overlay-panel.less

```diff
@@ -132,3 +149,7 @@ body.overlay-open {
     animation: modalSlideUp 0.2s ease-out forwards;
   }
 }
+
+.wizard-overlay .catalogs-overlay-panel {
+  padding-bottom: 0;
```

Comment:
I don't think you need to override here.  I think you just need to remove padding-bottom on .catalog-overlay-panel?